### DISCUSSION TITLE
Firmware libraries with problems in the include path

### DISCUSSION THREAD

**Discussion by Alanfis** [bitcraze/discussions#1197] [Q&A]
Hi, everyone.

I'm working with Crazyflie 2.1, and I want to embed the AHT10 sensor of the temperature and humidity in him, but the firmware does not recognize these libraries shown in the figure. The problem shown in VScode is related to "includePath", but I tried a lot of things to solve this and I didn't have success. I'm using VScode and Ubuntu 20.04. Somebody already had this problem and can help me?
![img](https://github.com/bitcraze/discussions/assets/16249420/ed12a327-2352-42ff-86c0-fc82c88071a1)

**Comment by ataffanel** [comment]
Do you also have the problem when actually compiling or is the problem only displayed by VSCode?

VSCode sometime gets out of sync with the project and does not find all the include files, so it is possible your code is actually compiling and this is just a false alert from VSCode. So can you check that things actually compile and if not, copy paste the compiler error message instead.

I think new version of VSCode can run Make in order to find the include folders, otherwise this is something you might have to configure manually.

**Reply by Alanfis** [reply]
Hi. This problem is only displayed by VScode. The drone recognizes the new firmware after I run the Make. Initially, I had written the wrong code, but then I corrected it and executed it correctly.
